packages/p2p/src/components/my-ads/buy-ad-payment-methods-list.jsx

```diff
@@ -0,0 +1,202 @@
+import * as React from 'react';
+import { Formik, Field } from 'formik';
+import { observer } from 'mobx-react-lite';
+import { Autocomplete, Icon } from '@deriv/components';
+import { useStores } from 'Stores';
+import { localize } from 'Components/i18next';
+import PropTypes from 'prop-types';
+
+const BuyAdPaymentMethodsList = ({ selected_methods, setSelectedMethods }) => {
+    const { my_ads_store, my_profile_store } = useStores();
+    const [selected_edit_method, setSelectedEditMethod] = React.useState();
+    const [payment_methods_list, setPaymentMethodsList] = React.useState(
```

Comment:
Can an empty [ ] be set as a default value here for the state?. since there is a `useEffect` hook that will compute and set the value when value of `selected_methods` change

Reply:
Yeap I'll set the default value to `[]`